release/Magarena/translations/español.txt

```diff
@@ -0,0 +1,726 @@
+¦1.87¦0
+1441438772 = es legal
+3128437712 = es ilegal
+3508543659 = está prohibida
+1998365398 = está restringida
+3534594353 = tiene demasiadas copias
+0137525990 = Tierra
+3541698722 = Criatura
+2979201658 = Artefacto
+1568509056 = Encantamiento
+3240984113 = Instantánea
+2797511102 = Conjuro
+3555281281 = Planeswalker
+0570309209 = Todas las cartas
+0295165923 = Aleatorio
+2313175938 = Listos para usar
+4274772376 = Mazos de jugador
+0506072877 = Mejores mazos de Firemind
+0233421099 = Mazos populares
+0439905227 = Mazos victoriosos
+2241583297 = Mazos jugados recientemente
+1120914852 = blancas
+2654390964 = azules
+2307242536 = negras
+4200685455 = rojas
+3499814433 = verdes
+2270227889 = Mano
+1849532704 = Biblioteca
+0454249737 = Cementerio
+1261234564 = Exilio
+1371036247 = Básica
+3795875175 = Común
+4148413569 = Poco común
+0696530817 = Rara
+2332676570 = Rara mítica
+3205463139 = Legendaria
+0975747761 = Ongoing
+2528184973 = Snow
+4223024711 = World
+2098643614 = Conspiracy
+1618575546 = Phenomenon
+0007471748 = Plane
+3092217981 = Scheme
+2421196991 = Tribal
+2154096770 = Vanguard
+0662641892 = Seleccione sí o no.
+3780066601 = Seleccione una carta de su mano.
+0033616675 = Seleccione %d cartas de su mano.
+1759655335 = Seleccione cara o cruz
+2805059655 = Seleccione un color.
+3462051701 = Seleccione a los atacantes.
+# 0979986174 eg. {f} will be replaced by an icon. | represents a new line.
```

Comment:
This line, in spanish, should be:
p. ej. {f} se reemplazará por un icono. | representa una nueva línea.